packages/bpk-component-datatable/src/BpkDataTableColumn-test.js

```diff
@@ -18,12 +18,15 @@
 
 import React from 'react';
 import { Column } from 'react-virtualized';
-import { spacingSm } from 'bpk-tokens/tokens/base.es6';
 
 import BpkDataTableColumn from './BpkDataTableColumn';
 import bpkHeaderRenderer from './bpkHeaderRenderer';
 
-const defaultProps = { label: 'Name', dataKey: 'name', width: spacingSm };
+const defaultProps = {
+  label: 'Name',
+  dataKey: 'name',
+  width: 12, // eslint-disable-line backpack/use-tokens
```

Comment:
why can't this still be `spacingSm`?

Reply:
Because `width` is defined as `number` in `react-virtualized` (https://github.com/bvaughn/react-virtualized/blob/master/source/Table/Column.js#L81), and using `spacingSm` throws a proptypes warning when running the tests :-(